lib/pos_snapshot_entries.go

```diff
@@ -995,15 +1013,18 @@ type LeaderPKIDAndIndex struct {
 	leaderPKID *PKID
 }
 
-func (bav *UtxoView) GetSnapshotLeaderSchedule() ([]*PKID, error) {
-	snapshotAtEpochNumber, err := bav.GetSnapshotEpochNumber()
+func (bav *UtxoView) GetCurrentSnapshotLeaderSchedule() ([]*PKID, error) {
+	snapshotAtEpochNumber, err := bav.GetCurrentSnapshotEpochNumber()
 	if err != nil {
-		return nil, errors.Wrapf(err, "GetSnapshotLeaderSchedule: problem calculating SnapshotEpochNumber: ")
+		return nil, errors.Wrapf(err, "GetCurrentSnapshotLeaderSchedule: problem calculating SnapshotEpochNumber: ")
 	}
+	return bav.GetSnapshotLeaderScheduleAtSnapshotAtEpochNumber(snapshotAtEpochNumber)
+}
+func (bav *UtxoView) GetSnapshotLeaderScheduleAtSnapshotAtEpochNumber(snapshotAtEpochNumber uint64) ([]*PKID, error) {
```

Comment:
I think it's perfectly fine to omit the second "Snapshot" in the function name since the "SnapshotLeaderSchedule" and the "snapshotAtEpochNumber" encapsulate what this function is intended for
```suggestion

func (bav *UtxoView) GetSnapshotLeaderScheduleAtAtEpochNumber(snapshotAtEpochNumber uint64) ([]*PKID, error) {
```